app/routes/health.js

```diff
@@ -1,13 +1,13 @@
 const routes = [{
   method: 'GET',
   path: '/healthy',
-  handler: (request, h) => {
+  handler: (_request, h) => {
     return h.response('ok')
   }
 }, {
   method: 'GET',
   path: '/healthz',
-  handler: (request, h) => {
+  handler: (_request, h) => {
```

Comment:
Since `request` isn't used, this is changed slightly to `_request` to make the intention explicit as suggested by SonarCloud.